Stratis.Bitcoin.Features.Consensus/StakeValidator.cs

```diff
@@ -300,12 +305,37 @@ public static ChainedBlock GetLastBlockIndex(StakeChain stakeChain, ChainedBlock
             return index;
         }
 
-        public static Target GetNextTargetRequired(StakeChain stakeChain, ChainedBlock indexLast, NBitcoin.Consensus consensus, bool proofOfStake)
+        /// <summary>
```

Comment:
it is supposed to be explained by the next sentence "so even a big change in the mining power on the network will be fixed by retargeting relatively quickly." 

What it means that the function of block time (spacing) is converging to the expected target spacing very quickly. So imagine your target spacing is 64 seconds and imagine your block time was 2000 seconds. If the function was slower, which could be done by trying to modify the target difficulty just by a bit to reach next target block time 10 seconds earlier, the function would go like this: 2000 -> 1990 -> 1980 -> 1970 ... and eventually reached 70 -> 64. That would be very slow. This happens when you add/subtract constant. What we do instead is that we multiply, which creates geometric decay - https://en.wikipedia.org/wiki/Exponential_growth and let's say we would only use constant factor of 10 % difference - i.e. 1.10x multiplier. It would go like this: 2000 -> 2000 / 1.1 = 1818 -> 1818 / 1.1 = 1652 -> 1652 / 1.1 = 1502 ... So it goes down much much faster. And this is almost what we do here except that the multiplier is not constant, but the overall exponential decay is preserved.